CHANGELOG.md

```
@@ -1,112 +1,109 @@
 
 ### Bug Fixes
 
-* fix: `home-page` key error in metadata (#45)
+- fix: `home-page` key error in metadata (#45)
 
-* Update __init__.py
+- Update **init**.py
 
 Replace:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)